tamr_client/backup.py

```diff
@@ -0,0 +1,120 @@
+from typing import List
+
+from tamr_client import Backup, response
+from tamr_client._types import Instance, JsonDict, Session, URL
+from tamr_client.exception import TamrClientException
+
+
+class InvalidOperation(TamrClientException):
+    """Raised when attempting an invalid operation.
+    """
+
+    pass
+
+
+class NotFound(TamrClientException):
+    """Raised when referencing a backup that does not exist on the server.
+    """
+
+    pass
+
+
+def _from_json(data: JsonDict) -> Backup:
+    """Make backup from JSON data (deserialize).
+
+    Args:
+        data: Backup JSON data from Tamr server
+    """
+    return Backup(
+        url=data["id"],
```

Comment:
The universal ID (the one starting with `unify://`), accessed here via `data["id"]`, is not a valid HTTP URL that we can query.

To remedy, the URL should be an argument to `_from_json` passed down by the caller.